internal/auth/ldap/repository_account.go

```diff
@@ -106,19 +108,90 @@ func (r *Repository) ListAccounts(ctx context.Context, withAuthMethodId string,
 	if err != nil {
```

Comment:
Can't comment on the right line, but don't forget to update the comment for this function (it supports both WithLimit and WithStartPageAfterItem)